ui/src/components/common/UploadControl.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { FormControl } from 'patternfly-react';
+
+class UploadControl extends Component {
+  localUpload = e => {
+    const { files } = e.target;
+    if (!files.length) {
+      return;
+    }
+
+    const file = files[0];
+    const start = 0;
+    const stop = file.size - 1;
```

Comment:
why do we deduct 1 here, if we add it later in the code?

Reply:
fixed